web_editor/js/main.js

```diff
@@ -6,17 +6,32 @@ import init, {
 } from "./osm2lanes-npm/osm2lanes_npm.js";
 
 await init();
-setupOnce();
+document.getElementById("start-editing").onclick = async function () {
+  try {
+    window.app = await LaneEditor.create();
+    window.app.render();
+  } catch (err) {
+    window.alert(`Error: ${err}`);
+  }
+};
 
 export class LaneEditor {
-  constructor(way, road_wrapper, locale, tags) {
-    const road = road_wrapper.Ok.road;
-
+  constructor(way, road, locale, tags) {
     this.way = way;
-    // Clone these
-    this.currentRoad = JSON.parse(JSON.stringify(road));
-    this.currentLocale = JSON.parse(JSON.stringify(locale));
+    this.road = road;
+    this.locale = locale;
     this.originalTags = tags;
+
+    // Remove these immediately. If they're rendered invisibly, they mess up indices.
```

Comment:
Still temporary. Seeing the center line where direction changes would be quite helpful visually.